src/components/internals/rich-text-body/rich-text-body.js

```diff
@@ -215,7 +222,7 @@ const RichTextEditorBody = React.forwardRef((props, ref) => {
             styles={tooltipStyles}
           >
             <Button
-              isActive={props.editor.hasUnderlinedMark()}
+              isActive={props.isFocused && props.editor.hasUnderlinedMark()}
```

Comment:
now that we are tracking the focus, we fix a UX problem and only display active buttons when the editor is in focus.